### ISSUE TITLE
Do this allow access to HomeKit devices via the Home App and this at the same time?

### ISSUE THREAD

**Issue by NorthernMan54** [mrstegeman/hap-controller-node#7]
I'm the developer behind the homebridge to Alexa integration and was looking at improving the integration, and the possibility of including real HomeKit devices.  In the README it wasn't clear and unfortunately I don't have any real HomeKit devices in my setup ( strictly homebridge for everything ) to try it out.

My hope is that this will allow dual access, from both the Home App and the API as well.

**Comment by mrstegeman** [comment]
The answer to this is... complicated. The HAP spec defines a way to add and remove pairings, so that access can be shared between multiple devices. However, it doesn't define what the process is for sharing the device identifier with the master device.

This library does have the required methods to [add](https://github.com/mrstegeman/hap-controller-node/blob/master/lib/transport/ip/http-client.js#L177-L197) and [remove](https://github.com/mrstegeman/hap-controller-node/blob/master/lib/transport/ip/http-client.js#L157-L175) pairings. I'm just not sure how to either get an iOS device to share its identifier, or how to share our identifier with the iOS device.

tl;dr: No, you have to use either the Home app or this library.

**Comment by NorthernMan54** [comment]
Tks very much for the detailed response.
